drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java

```diff
@@ -273,15 +275,23 @@ public static Point2D getMiddleXYCell(final GridWidget gridWidget, final GridCol
     }
 
     public static String getPlaceholder(String canonicalClassName) {
-        return isLocalDate(canonicalClassName) ?
-                ScenarioSimulationEditorConstants.INSTANCE.dateFormatPlaceholder() :
-                ScenarioSimulationEditorConstants.INSTANCE.insertValue();
+        if (isLocalDate(canonicalClassName)) {
```

Comment:
@jomarko 
Hi - Maybe this could be replaced with switch - it was not so because there where just two possibilities, but if adding more is needed, the switch is more easier to read and extend in future (the called methods just compares Strings, and since for each comparison one of the compared term is a final static, I think they can be used in the case statement)

Reply:
ok, will do